internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -12,8 +12,9 @@ import (
 )
 
 type KafkaInstanceType struct {
-	Id    string              `yaml:"id"`
-	Sizes []KafkaInstanceSize `yaml:"sizes"`
+	Id          string              `yaml:"id"`
+	DisplayName string              `yaml:"display_name"`
```

Comment:
Is `display_name` a required parameter? Do we want to add validation for this to ensure it's specified?

Reply:
done. let me know if that's sufficient

Reply:
looks good, thanks!